### PR TITLE
feat/ignore: adds *ignore files

### PR DESCRIPTION
**Pull request by xanimo**
.dockerignore to omit unnecessary and potentially sensitive files from being added into our image.
.gitignore to omit pycache and generated python test files

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.git
+.github
+.gitignore
+.cache
```
Comment (patricklodder) [inline .dockerignore:4]: this probably needs to go into `.gitignore` too?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+1.14.5/
+tests/
+tools/
```
Comment (patricklodder) [inline .dockerignore]: why?

Comment (patricklodder) [inline .dockerignore]: please add newline

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*.py[cod]
+__pycache__/
```
Comment (patricklodder) [inline .gitignore]: ignore also in `.dockerignore`?

Comment (patricklodder) [inline .gitignore]: please add newline

**Review by patricklodder — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (xanimo) [reply .dockerignore:4]: done

File: .dockerignore (hunk 2 above)
Comment (xanimo) [reply .dockerignore]: removed. disregard for now, i need to do more research.

Comment (xanimo) [reply .dockerignore]: hmm i added a newline on both and for some reason they're not reflecting on either?

File: .gitignore (hunk 3 above)
Comment (xanimo) [reply .gitignore]: done

**Review by patricklodder — APPROVED** [review]

**Review by AbcSxyZ — APPROVED** [review]

**Pull request merged**